app/src/main/java/com/example/theperiodpurse/ui/calendar/CalendarScreen.kt

```diff
@@ -303,19 +331,27 @@ fun CalendarScreenLayout(navController: NavController) {
                         .semantics { contentDescription = "Calendar" },
                     state = state,
                     monthHeader = { month ->
-                        MonthHeader(month) },
+                        MonthHeader(month)
+                    },
                     dayContent = { day ->
-                        Day(day, isSelected = selectedDate == day.date) { date ->
-                            selectedDate = if (selectedDate == date.date) null
-                            else date.date
-                            navController.navigate(
+                        (if (activeSymptom == null) Symptom.FLOW else activeSymptom)?.let {
```

Comment:
Instead of having the `activeSymptom` being nullable, can we have simply default it to `Symptom.FLOW`? In the IOS version, `Symptom.FLOW` is always in the tab and you cannot remove it, so we should never have a situation where 'activeSymptom' is 'null'. This should reduce some checking.

Reply:
Yes, that makes it a lot cleaner. I've updated it to default to `Symptom.FLOW`